nvr/nvr.py

```diff
@@ -297,17 +309,27 @@ def split_cmds_from_files(args):
 def prepare_filename(fname):
     return os.path.abspath(fname).replace(" ", "\ ")
 
+    
+def get_sockaddrs_for_printing(proc):
+    sockaddrs = []
+    for conn in proc.connections('inet4'):
+            sockaddrs.insert(0, ':'.join(map(str, conn.laddr)))
+            
+    if sys.platform == "win32":
+        sockaddrs += ["win32 pipe: {}".format(pipename) for pipename in os.listdir('\\\\.\\pipe')]
+    else:
+        for conn in proc.connections('unix'):
+            if conn.laddr:
+                sockaddrs.insert(0, conn.laddr)
+            
+    return sockaddrs
+
 
 def print_sockaddrs():
     sockaddrs = []
 
     for proc in psutil.process_iter():
-        if proc.name() == 'nvim':
```

Comment:
The `proc.name() == 'nvim'` check was dropped.  Was that intentional?  Won't this now show information for all processes on the system?